kolena/dataset/dataset.py

```diff
@@ -258,8 +258,16 @@ def _send_upload_dataset_request(
     load_uuid: str,
     sources: Optional[List[Dict[str, str]]],
     append_only: bool = False,
+    commit_tags: Optional[List[str]] = None,
```

Comment:
Intentionally not adding the param to the public function as the FE of this piece hasn't been planned yet